src/Microsoft.Dnx.Tooling/Restore/RestoreOperations.cs

```diff
@@ -27,42 +27,39 @@ public RestoreOperations(IReport report)
             sw.Start();
 
             var dependencies = new List<LibraryDependency>();
-            if (context.RuntimeSpecs != null && !string.IsNullOrEmpty(context.RuntimeName))
-            {
-                // HACK(davidfowl): This is making runtime.json support package redirects
 
-                // Look up any additional dependencies for this package
-                foreach (var runtimeSpec in context.RuntimeSpecs)
+
+            if (context.RuntimeSpecs != null && !string.IsNullOrEmpty(context.RuntimeName) && context.RuntimeDependencies != null)
```

Comment:
Can `context.RuntimeDependencies` be ever null? Looks like it is always assigned and can only be an empty dictionary.